library/cephadm_adopt.py

```diff
@@ -122,6 +123,35 @@ def main():
 
     startd = datetime.datetime.now()
 
+    cmd = ['cephadm', 'ls', '--no-detail']
+
+    if module.check_mode:
+        exit_module(
+            module=module,
+            out='',
+            rc=0,
+            cmd=cmd,
+            err='',
+            startd=startd,
+            changed=False
+        )
+    else:
+        rc, out, err = module.run_command(cmd)
+
+    if rc == 0:
+        if name in [x["name"] for x in json.loads(out) if x["style"] == "cephadm:v1"]:
```

Comment:
if rc == 0, x['style'] is always defined, right?

Reply:
AFAIK the `style` key is always defined either with `legacy` or `cephadm:v1` value

Reply:
"ok, sounds good" :grin: